source/help/apps/desktop-changelog.rst

```diff
@@ -60,6 +60,11 @@ Mac
 
 Known Issues
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- When minimizing the app and then maximizing it, it doesn't restore to the right position but "jumps" to a different place in the taskbar. [MM-23195](https://mattermost.atlassian.net/browse/MM-23195)
+- The current window frame and server tabs don't work properly on older Windows OS versions or Linux. [MM-22751](https://mattermost.atlassian.net/browse/MM-22751)
```

Comment:
is this an issue or a feature?

Reply:
It's a feature ticket but it has been reported by lot of users after v4.4 was released.

Reply:
@amyblais can we be more specific about "doesn't work properly"

It works fine, it's just not styled consistently with the rest of the OS in Windows 7 or Linux